pkg/apis/apps/types.go

```diff
@@ -61,6 +61,18 @@ const (
 	// replica count is changed, and will not wait for pods to be ready or complete
 	// termination.
 	ParallelPodManagement = "Parallel"
+	// RollingUpdateStatefulSetStrategyType indicates that update will be
```

Comment:
please put these consts back where they were... they are actually supposed to be of type StatefulSetUpdateStrategyType, which is a different issue, but putting them here is confusing and incorrect